langchain/chat_models/openai.py

```diff
@@ -372,10 +372,11 @@ def get_num_tokens(self, text: str) -> int:
         enc = tiktoken.encoding_for_model(self.model_name)
 
         # encode the text using the GPT-3.5-Turbo encoder
-        tokenized_text = enc.encode(text)
+        return enc.encode(text)
 
-        # calculate the number of tokens in the encoded text
-        return len(tokenized_text)
+    def get_num_tokens(self, text: str) -> int:
```

Comment:
I think this would be unnecessary now since it would call the base class, right?